external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.kafka010
 
 import java.{util => ju}
+import java.time.{Duration => jDuration}
```

Comment:
This will avoid conflicts with `import scala.concurrent.duration.Duration` at line 27.